easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.0.1-foss-2016a.eb

```diff
@@ -0,0 +1,26 @@
+name = 'OpenFOAM'
+version = '2.0.1'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package. 
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer, 
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://downloads.sourceforge.net/foam/%(version)s']
+sources = [
+    'OpenFOAM-%(version)s.gtgz',
+    'ThirdParty-%(version)s.gtgz',
+]
+
+patches = [
+    'cleanup-OpenFOAM-%(version)s.patch',
+    'cleanup-OpenFOAM-2.0.1-GCC49.patch',
```

Comment:
when adding new patches, please name them `OpenFOAM-%(version)s_<label>.patch`

existing patches for older versions of OpenFOAM (as is the case here) can be left as-is